### ISSUE TITLE
How to prepare pytorch model weight?

### ISSUE THREAD

**Issue by RorschachChen** [gatheluck/FourierHeatmap#14]
I tried to train ResNet18 from torchvision on CIFAR10 from scratch and with a 90+% accuracy, but when I evaluate the fourier heatmap of my model weight, I found that the result is not like the example.  Do you have any idea about the reason or how can I get the pretrained model weight properly? Thanks for replying.
![resnet18_cifar10_fourier_heatmap](https://user-images.githubusercontent.com/36701362/167242645-bd03b9b0-beb4-477e-8dd9-c32795e523db.PNG)
